File: Easy/Python 3/Move_Zeroes(283).py
class Solution:
    def moveZeroes(self, nums: List[int]) -> None:
        """
        Do not return anything, modify nums in-place instead.
        """
        shift_count = 0
        for index in range(len(nums)):
            if nums[index] != 0:
                nums[shift_count] = nums[index]
                shift_count += 1
    
        for index1 in range(shift_count, len(nums)):
            nums[index1] = 0

File: Easy/Java/Move_Zeroes(283).java
class Solution {
    public void moveZeroes(int[] nums) {
	int shift_count = 0;
	for (int i = 0; i < nums.length; i++) {
		if (nums[i] != 0) {
			nums[shift_count++] = nums[i];
		}
	}
	for (int i = shift_count; i < nums.length; i++) {
		nums[i] = 0;
	}
    }
}